wrappers/src/shared_realm_cs.cpp

```diff
@@ -48,7 +48,8 @@ using namespace realm::util;
 using OpenRealmCallbackT = void(void* task_completion_source, ThreadSafeReference* ref, NativeException::Marshallable ex);
 using RealmChangedT = void(void* managed_state_handle);
 using ReleaseGCHandleT = void(void* managed_handle);
-using LogMessageT = void(realm_string_t message, util::Logger::Level level);
+// TODO(lj): Update arg order to be the same across the SDK.
+using LogMessageT = void(realm_string_t message, util::Logger::Level level, realm_string_t category_name);
```

Comment:
Will likely update this to use the same arg order as the SDK's `Logger.Log()` for consistency (level, message, category), even though having the message in between looks somewhat odd.